### PR TITLE
Fix Iris/Sigmet, align with CfRadial2

### PR DESCRIPTION
**Pull request by kmuehlbauer**
This fixes the issue with #51 and removes some inconsistencies with the common CfRadial2-based API. Also undefaulted angle reindexing and removed `a1gate` and `angle_res` from angle-coordinate.

ping @aladinor @mgrover1

### CODE REVIEW

File: xradar/io/backends/iris.py [hunk 1]
```diff
@@ -3665,9 +3665,15 @@ def get_data_headers(self):
     def site_coords(self):
         """Return Radar Location Tuple"""
         ing_conf = self.ingest_header["ingest_configuration"]
+        lon = ing_conf["longitude_radar"]
+        lat = ing_conf["latitude_radar"]
+        lon = lon if lon <= 180 else lon - 360
+        # todo: is this correct for southern latitudes?
+        lat = lat if lat <= 180 else lon - 360
```
Comment (mgrover1) [inline xradar/io/backends/iris.py:3672]: I don't think this is right the logic for southern latitudes. Latitude will be between 0 and -90 in the southern hemisphere

**Review by mgrover1 — COMMENTED** [review]
See the one minor comment

File: xradar/io/backends/iris.py (hunk 1 above)
Comment (kmuehlbauer) [reply xradar/io/backends/iris.py:3672]: So, I was thinking the following:

- the output from iris/sigmet file is between 0 and 360 (binary angle decoding)
- the latitude runs from equator (0deg) to the north pole (90deg) and over back to the equator again (180deg, -180deg)
- then latitude runs to the south pole (270deg, -90deg) and up to equator again (0deg)

So with this assumption this formulation would be correct although it looks very weird. I've checked with Py-Art, the same formulation is used there. But no hints in the code.

Comment (mgrover1) [reply xradar/io/backends/iris.py:3672]: Ahh okay! I am not as familiar with this format... that sounds reasonable then! Sorry for the confusion!

Comment (kmuehlbauer) [reply xradar/io/backends/iris.py:3672]: No worries, I'm first convinced that this is correct, if I have either proof from file or from Vaisala :grimacing:

Comment (mgrover1) [reply xradar/io/backends/iris.py:3672]: Sounds good!! Yeah - holding off on merging that Py-ART PR until we get the confirmation.

**Review by mgrover1 — APPROVED** [review]
Approved assuming all is good with Vaisala :)

**Comment by kmuehlbauer** [comment]
> Approved assuming all is good with Vaisala :)

We can still change later, if it turns out other way.

**Pull request merged**